torch/_torch_docs.py

```diff
@@ -2509,6 +2509,12 @@ def merge_dicts(*dicts):
 Keyword args:
     {out}
 
+.. note::  This function is similar to SciPy's `scipy.special.digamma`.
+
+.. note::  From version 1.8 onwards, the digamma function returns `NaN` for non-positive integers,
```

Comment:
We should probably get in the habit of just saying "From PyTorch 1.8..." instead of "version" since "version" could be ambiguous.